include/wlr/config.h.in

```diff
@@ -15,4 +15,9 @@
 
 #mesondefine WLR_HAS_XDG_FOREIGN
 
+#mesondefine WLR_HAS_GBM_SUPPORT
+#mesondefine WLR_HAS_GLES2_RENDERER
+
+#mesondefine WLR_HAS_DRM_BACKEND
+#mesondefine WLR_HAS_LIBINPUT
```

Comment:
This should be named `WLR_HAS_LIBINPUT_BACKEND` for consistency with the rest of the backends.

Reply:
`WLR_HAS_LIBSEAT`
There is a library that inside backend session, the only reason I made BACKEND suffix is that those macro are only used
for backend. Well if you want that, I can agree with you, but it just make that macro look too so long.

Reply:
I'd prefer to have the `_BACKEND` prefix for consistency.